src/views/Swap/components/Chart/styles.tsx

```diff
@@ -16,3 +15,7 @@ export const StyledPriceChart = styled(Box)<{ $isDark: boolean; $isExpanded: boo
     height: ${({ $isExpanded }) => ($isExpanded ? 'calc(100vh - 100px)' : '484px')};
   }
 `
+
+StyledPriceChart.defaultProps = {
```

Comment:
Do we need to define defaultProps here? Seems like its always passed explicitly

Reply:
make it can be pass props and override this height here:
https://github.com/pancakeswap/pancake-frontend/pull/2790/files#diff-ed213cc974f6038fe2fa01c988709f3d5fed366037820f8297fc0ca297dc9a97R48-R49